ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -75,6 +75,26 @@ The following ansible plugins have been removed from the `arista.avd` Ansible co
 
 TODO: Level 3 sections for each change with details on how to migrate
 
+### Removed `redistribute igmp` from BGP vlan config belonging to a VRF with evpn multicast
```

Comment:
This is not really a porting guide entry. We need to describe the change (in words or a config snip. not a structured config snip) and how to revert the change if you prefer to.
So we need to show how the user can keep the redistribute igmp. If we do not have a knob for that, we should add that.

Reply:
Added a knob `redistribute_igmp_vrf`